src/app/core/components/admin/admin.interceptor.ts

```diff
@@ -28,10 +30,16 @@ export class AdminInterceptor implements HttpInterceptor {
     } else {
       this.authService = this.userAuthService;
     }
-    if (this.authService.getAccessToken()) {
+
+    if (this.isRefreshing) {
+      request = this.addToken(request, this.authService.getRefreshToken());
+    } else if (this.authService.getAccessToken()) {
```

Comment:
Maybe will be better set "this.authService.getAccessToken()" to some variable?

Reply:
Why? I use "this.authService.getAccessToken()" only once